Description: Documentation fixes
 This patch includes fixes to the XML documentation, some of which enable the
 processing tools to compile the documentation from XML, and make it DTD
 compliant.
Author: Roland Stigge <stigge@antcom.de>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=516237

--- dia-0.97.1.orig/doc/en/usage-layers.xml
+++ dia-0.97.1/doc/en/usage-layers.xml
@@ -41,9 +41,15 @@
     <para>
       You can use layers to create overlay slide show by drawing each incremental slide in a
       layer and manipulating which layers are visible. If you want to automate
-      post processing such slide show, you can use <command>--show-layers</command> command line switch.
+      post processing such slide show, you can use --show-layers command line switch.
       For example: 
-      <command>dia --show-layers=Background,Slide3 --filter=eps-builtin --export=foo.eps foo.dia</command>
+       <cmdsynopsis>
+       <command>dia</command>
+       <arg choice='plain'>--show-layers=Background,Slide3</arg>
+       <arg choice='plain'>--filter=eps-builtin</arg>
+       <arg choice='plain'>--export=foo.eps</arg>
+       <arg choice='plain'>foo.dia</arg>
+       </cmdsynopsis>
       will export the Background and Slide3 layers, i.e. the eps output has only these
       two layers and none others. You would then repeat the export for each
       combination of layers you need to produce your slides. Using this switch
--- /dev/null
+++ dia-0.97.1/doc/pl/entities.xml
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+
+<!-- e-novative DocBook environment -->
+<!-- local enties                   -->
+<!-- (c) 2002 e-novative GmbH       -->
+<!-- http://www.e-novative.de       -->
+
+<!ENTITY local_entity "local entitiy (defined per document)">
--- dia-0.97.1.orig/doc/pl/dia.xml
+++ dia-0.97.1/doc/pl/dia.xml
@@ -1,5 +1,5 @@
-<?xml version="1.0" encoding='iso-8859-2' ?>
-<!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook V4.5//EN" "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd"
+<?xml version="1.0" encoding="iso-8859-2" ?>
+<!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN" "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd"
 
 [
 <!ENTITY VERSION "0.92.2">
@@ -16,6 +16,8 @@
 <!ENTITY AUTHORS-NOTE SYSTEM "authors-note.xml">
 <!ENTITY AUTHORS SYSTEM "authors.xml">
 <!ENTITY LICENSE SYSTEM "license.xml">
+<!ENTITY % local_entities SYSTEM "entities.xml">
+%local_entities;
 ]>
 
 <book id="index"> <!-- proszê nie zmieniaæ identyfikatora -->
